### ISSUE TITLE
Coders-Mexico Democratico

### ISSUE THREAD

**Issue by josellausas** [codersmexico/codersmexico#16]
- **Como** _desarrollador_ **quiero** crear un sistema de votaciones auditable y transparente **para** poder experimentar con una democracia asitida por la tecnología dentro de la comunidad.

- **Como** _admin_ de la comunidad **quiero** organizar al grupo para que experimentemos con nuevas formas de organización democrática **para** enconrtar nuevas soluciones a problemas enormes.

- **Como** _académico_ **quiero** ayudar a diseñar el experimento social y desarrollar un paper **para** documentar los conocimientos adquiridos.

**Comment by eruizdechavez** [comment]
Yo tengo 0 experiencia escribiendo o diseñando estudios/experimentos sociales, pero me interesa en el area de la implementación.

**Comment by josellausas** [comment]
@eruizdechavez Estaría bien invitar a alguien que le sepa profesionalmente a esos temas.

**Comment by eruizdechavez** [comment]
Del lado tecnico, creo que podríamos aprender mucho si nos enfocamos en diferentes cuestiones de usabilidad y accesibilidad, por ejemplo, incluir temas de accesibilidad (accessibility, A11y) y medios alternos de votación como respuesta por correo electrónico o SMS.

**Comment by diegoaguilar** [comment]
Me parece una idea muy interesante. Si se comienza me interesa cooperar con el login con Amazon Rekognition usando el rostro del usuario (en caso de que exista esa interfaz de acceso/login). 

Estoy interesado en probar Rekognition asap, that's why

**Comment by eruizdechavez** [comment]
@diegoaguilar suena interesante pero no se que tan accesible sea. Asumo que necesitarías tener una computadora con cámara web y buena conexión para usarlo; en cierta medida creo que estaría restringiendo el uso a quien tenga dicha tecnología/recursos/conocimientos para usarla.

**Comment by diegoaguilar** [comment]
Sí, de acuerdo ... Por ahí iba el _(en caso de que exista esa interfaz de acceso/login)._

**Comment by Tecayehuatl** [comment]
¿Hay algún prototipo o algo? Suena interesante.

**Comment by eruizdechavez** [comment]
@Tecayehuatl hasta donde se, no, nos enfocamos primero en otras tareas de mantenimiento del Slack, como la limpieza de canales
